src/components/BulkOperations/BulkOperations.scss

```diff
@@ -0,0 +1,27 @@
+.bulk-operations-spinner {
+    min-height: 65vh;
```

Comment:
[curious] any reason for increasing the size of spinner?

Reply:
Otherwise the footer renders too close to the top, which looks odd

Reply:
I think it is fine (footer moving above during loading).

Reply:
The Course Pages have the same behavior (footer stays down) too.